src/shortcuts/redo.ts

```diff
@@ -1,20 +1,33 @@
-import { Dispatch } from 'react'
 import RedoIcon from '../components/RedoIcon'
-import { Shortcut } from '../@types'
+import { Patch, Shortcut } from '../@types'
 import { isRedoEnabled } from '../selectors/isRedoEnabled'
+import { alert as alertAction } from '../action-creators'
+import { NAVIGATION_ACTIONS } from '../constants'
 
-interface RedoAction {
-  type: 'redoAction'
+/**
+ * Recursively calculates last action type from patches history if it is one of the navigation actions and finally returns the action.
+ */
+const getActionTypeRecursively = (patches: Patch[], n = 1): string => {
```

Comment:
I suggest removing "Recursively" from the name. Function names should not include reference to implementation details.

Reply:
Sure, will do!